api/orchestrator/orchestrator.proto

```diff
@@ -949,6 +963,28 @@ message RemoveCertificateRequest {
   string certificate_id = 1 [ (validate.rules).string.min_len = 1 ];
 }
 
+message Runtime {
+  // release_version is the latest Clouditor release version for this commit
+  string release_version = 1;
```

Comment:
should this be set to optional, since this is only used for release versions? I guess in non-release versions this is empty?